axelrod/tests/strategies/test_axelrod_first.py

```diff
@@ -138,6 +138,31 @@ def test_strategy(self):
         self.versus_test(axelrod.Defector(), expected_actions=actions)
 
 
+class TestGraaskamp(TestPlayer):
+
+    name = "Graaskamp"
```

Comment:
`Graaskamp: 0.05` (the parameters get automatically added).

Reply:
I'm not sure what should I do here. Should I add parameters manually somehow for testing? Thanks :)

Reply:
Yes, you can change the test name to "Graaskamp: 0.05"